src/battle_script_commands.c

```diff
@@ -11060,7 +11096,7 @@ static void Cmd_pursuitrelated(void)
         gCurrentMove = MOVE_PURSUIT;
         gBattlescriptCurrInstr += 5;
         gBattleScripting.animTurn = 1;
-        gBattleScripting.field_20 = gBattlerAttacker;
+        gBattleScripting.savedBattler = gBattlerAttacker;
```

Comment:
I see you renamed field 20, but have not changed the name in constants/battle_script_commands.h. Also, are you sure it doesn't collide with Pursuit?

Reply:
Ooh, you're right, I didn't rename it in that file - that's fixed. That field is only used by Pursuit, as far as I can tell, so I updated that move accordingly.